src/tink/http/containers/NodeContainer.hx

```diff
@@ -72,6 +97,44 @@ class NodeContainer implements Container {
           },
           failures: failures,//TODO: these need to be triggered
         }));
-      });      
+      }
+      
+      if(untyped server.listening) // .listening added in v5.7.0, not added to hxnodejs yet
```

Comment:
If you dare to use the `github` version of `hxnodejs`:
https://github.com/HaxeFoundation/hxnodejs/pull/104#event-1723257365

Reply:
Great, but I think I will keep it as-is until hxnodejs got a proper release. So not to frustrate tink_http users.

Reply:
Sure, makes sense. I still haven't got time to test my project with this.